### DISCUSSION TITLE
Window auto closing/script not running

### DISCUSSION THREAD

**Discussion by Tyemnata** [UltimaHoarder/UltimaScraper#1817] [Q&A]
Hello, I had been using 7.4.1 and updated to the latest version of this script and 10.1 of Python and started from a fresh directory. I start the script, select 0 for OF, select all for username, get the welcome message then get a flash of lines in the console window and the window closes itself. It's literally a flash and I can't even get a screenshot of what it says. Is there a log for the script or python I can look at to try to debug this?

**Comment by jozipaulo** [comment]
This is an old version of the scraper program. The latest is 7.6.1 at this time (26 Dec 2021)
You can locate the latest version here - https://github.com/DIGITALCRIMINALS/OnlyFans
Looks like this DIGITALCRIMINAL not DIGITALCRIMINALS repository is no longer active.

**Reply by Tyemnata** [reply]
Ah, I see that now. I still have the same issue, but I ran the script from an existing terminal so I have the error now. I will Post my issue on the new page.
